src/quadprog.jl

```diff
@@ -1,13 +1,74 @@
 export quadprog
 
+const MatrixOrOperator = Union{AbstractMatrix, AbstractLinearOperator}
```

Comment:
This could be put inside `auxiliary.jl`, and removed from here and from `linprog`.